hdr_approx_percentile.sql

```diff
@@ -0,0 +1,232 @@
+-- --------
+-- Overview
+-- --------
+--
+-- This is a demo test to show how to use hdr_histogram (http://hdrhistogram.org/)
+-- to compute `approx_percentile` in the RisingWave stream engine.
+--
+-- The advantage of this approach is that it supports updatable approx_percentile.
+-- The disadvantage is that it requires quadratic computation and storage
+-- as the precision increases,
+-- due to streaming nested loop join.
+--
+-- The way it works is that hdr_histogram stores N significant digits of each value,
+-- thereby reducing the number of buckets needed to store the distribution.
+--
+-- ----------------------
+-- Implementation details
+-- ----------------------
+--
+-- First we create the histogram itself, where each value is encoded as a triple:
+-- 1. The sign of the value (1 or -1)
+-- 2. The exponent of the value (the power of 10, truncated to an integer)
```

Comment:
Is 10 the only choice here? 
It seems also affect how approximated the percentile would be in the end

Reply:
Nope, can use 2 also. But I choose 10 because our system only has `log10`. `log` seems to be an alias for `log10` as well.